### PR TITLE
Add values for registry storage back end

### PR DESCRIPTION
**Pull request by ianstanton**
* Add value `gcp_default_service_account_key` to `astronomer` module
* Add value `container_registry_bucket_name` to `astronomer` module
* Related to astronomer/biz-cloud#72

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -22,7 +22,9 @@ module "astronomer" {
...
   private_load_balancer = false
   # indicates which kind of LB to use for Nginx
-  cluster_type  = "public"
-  enable_istio  = "true"
-  enable_gvisor = "true"
+  cluster_type                    = "public"
+  enable_istio                    = "true"
+  enable_gvisor                   = "true"
+  gcp_default_service_account_key = module.gcp.gcp_default_service_account_key
+  container_registry_bucket_name  = module.gcp.container_registry_bucket_name
```
[the remark this answers is not in the document]
Comment (schnie) [reply main.tf]: Makes sense. Separately, should we be specifying `"private"` for `cluster_type`? Or are we overriding that with tfvars for our actual deployment?

[the remark this answers is not in the document]
Comment (ianstanton) [reply main.tf]: That should be set in astronomer/terraform-google-astronomer-cloud I believe. @sjmiller609 is that correct? astronomer/terraform-kubernetes-astronomer#9 has been merged.

**Comment by ianstanton** [comment]
Think there are some new conflicts here. I'll look into that.

**Comment by schnie** [comment]
Looks good to me, other than the "public" cluster. This seems like where we'd override that to be "private". What's going on there?

Also looks like there's some merge conflicts now.

File: main.tf (hunk 1 above)
[the remark this answers is not in the document]
Comment (sjmiller609) [reply main.tf]: I don't think that variable exists any more

[the remark this answers is not in the document]
Comment (schnie) [reply main.tf]: So it is now https://github.com/astronomer/terraform-google-astronomer-gcp/blob/master/variables.tf#L91-L93 and private by default right?

**Comment by ianstanton** [comment]
I believe I'll need to add `gcp_default_service_account_key` as an output in `terraform-google-astronomer-gcp`

**Comment by ianstanton** [comment]
@sjmiller609 it looks like that output was removed https://github.com/astronomer/terraform-google-astronomer-gcp/commit/0dbabafa33dc248524abe86044d2b1dd177846d9#diff-c09d00f135e3672d079ff6e0556d957dL43 What was the reason for this?

**Comment by sjmiller609** [comment]
oops, I'll throw that back in

**Comment by sjmiller609** [comment]
https://github.com/astronomer/terraform-google-astronomer-gcp/pull/18

File: main.tf [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module "gcp" {
-  source              = "astronomer/astronomer-gcp/google"
-  version             = "1.0.15"
+  source  = "astronomer/astronomer-gcp/google"
+  version = "1.0.15"
```
Comment (sjmiller609) [inline main.tf]: 1.0.16 fixes the issue where I deleted your code :)

Comment (ianstanton) [reply main.tf]: Thanks!

File: main.tf (hunk 1 above)
[the remark this answers is not in the document]
Comment (sjmiller609) [reply main.tf]: that's right - and nodes are always private. https://github.com/astronomer/terraform-google-astronomer-gcp/blob/master/main.tf If you do "public" for management_api, then it whitelists your IP address

**Comment by ianstanton** [comment]
@sjmiller609 Anything else we need to add here? Good to merge?

**Pull request merged**